saleor/graphql/product/sorters.py

```diff
@@ -45,10 +53,10 @@ def description(self):
                 "Sort attributes by their position in storefront"
             ),
             AttributeSortField.DASHBOARD_VARIANT_POSITION.name: (
-                "Sort variant attributes by their position in dashboard."
+                "Deprecated: Sort variant attributes by their position in dashboard."
```

Comment:
For any deprecation notice, we should say in which version something will be removed. Since we have now 2.9 and removed stuff 2 versions later, it should say: "DEPRECATED: Will be removed in Saleor 2.11." Same below.

Reply:
sure

Reply:
done